### PR TITLE
Updated the documentation for release 4.3.0

### PR DESCRIPTION
**Pull request by hardikhdholariya**
- Updated the documentation

### CODE REVIEW

**Comment by hardikhdholariya** [comment]
Few Screenshots needs to update. I'll update after we replace the device inventory.

**Comment by VatsalJagani** [comment]
Looks good to me. But we'll merge this PR just before the release.

File: docs/release_notes/release_notes.md [hunk 1]
```diff
@@ -9,46 +9,60 @@ has_children: true
 # Release Notes
 
 
-## Version 4.2.0/4.2.1 (August 2023)
+## Version 4.3.0 (September 2023)
```
Comment (VatsalJagani) [inline docs/release_notes/release_notes.md:12]: Change this to October

File: docs/release_notes/release_notes.md [hunk 2]
```diff
@@ -9,46 +9,60 @@ has_children: true
...
-    * Added a generic search filter in the Forensics and SOC dashboards.
-
-    * Excluded Windows password expired events from the alerts related to brute force.
+    * #### Device Inventory
+        * Enhanced the device inventory logic by: 
+            * Updating the device matching and hostname matching logic.
+            * Modifying its data structure.
+            * Adding the functionality to auto merge devices.
+            * Updating the functionality to manual merge devices.
+        * Updated the following dashboards by updating its searches:
```
Comment (VatsalJagani) [inline docs/release_notes/release_notes.md:32]: Do not have to list this.

**Review by VatsalJagani — APPROVED** [review]

**Pull request merged**